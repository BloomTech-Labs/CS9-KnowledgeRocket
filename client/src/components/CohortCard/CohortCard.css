.CohortCard_Form {
	width: 100%;
}
