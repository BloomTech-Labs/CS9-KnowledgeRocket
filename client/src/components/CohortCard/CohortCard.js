import React, { Component } from 'react';
// import { connect } from 'react-redux';
// Material Components
import CardContent from '@material-ui/core/CardContent';
import Input from '@material-ui/core/CardContent';
import FormControl from '@material-ui/core/FormControl';

// function mapStateToProps(state) {
//     return {

//     };
// }

// RENDERS A COHORT CARD FOR A SINGLE CLASS - receives class info from CohortList
class CohortCard extends Component {
	render() {
		// each card renders:
		// Cohort name
		// Number of students
		// Participation(%)
		// Rockets sent
		return (
			<CardContent>
				<FormControl>
					<Input>Name:</Input>
					<Input>Number of Students:</Input>
					<Input>Participation:</Input>
					<Input>Rockets Sent:</Input>
				</FormControl>
			</CardContent>
		);
	}
}

export default CohortCard;
// export default connect(mapStateToProps)(CohortCard);
