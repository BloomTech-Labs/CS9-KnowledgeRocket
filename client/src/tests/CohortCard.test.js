import React from 'react';
import renderer from 'react-test-renderer';
// Component
import CohortCard from '../components/CohortCard/CohortCard';

describe('CohortCard', () => {
	it('renders correctly', () => {
		const tree = renderer.create(<CohortCard />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
