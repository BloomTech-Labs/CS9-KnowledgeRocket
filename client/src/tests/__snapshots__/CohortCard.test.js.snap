// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CohortCard renders correctly 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1"
>
  <div
    className="MuiCardContent-root-29"
  >
    <div
      className="MuiFormControl-root-30"
    >
      <div
        className="MuiCardContent-root-29"
      >
        <a
          className="sc-bdVaJa iqJYxZ"
          href="/rocket/classForm/123"
          onClick={[Function]}
        >
          Calculus 101
        </a>
      </div>
      <div
        className="MuiCardContent-root-29"
      >
        Students:
         
        3
      </div>
      <div
        className="MuiCardContent-root-29"
      >
        Participation:
      </div>
      <div
        className="MuiCardContent-root-29"
      >
        Rockets Sent:
      </div>
    </div>
  </div>
</div>
`;
